### ISSUE TITLE
Redundant aria-label on the svg for expand/collapse in clr-stack-view

### ISSUE THREAD

**Issue by immihaylova** [vmware/clarity#4525]
This is an issue with accessibility. We got a report from AccessLevel where they note that the expand/collapse icon inside a clr-stack-view has a redundant aria-label.

Here is the issue described by them:

> Although technically nothing wrong with the code in the screenshot, the issue with these accordions is that the parent div with the role=”button” has attribute aria-expanded which already conveys the state. Adding the textual equivalent (“Collapse” or “Expand”) to the svg images may be redundant and can be verbose to screen reader users. The functionality of collapsing and expanding is set on the div with the role=”button” and not on the image itself which is why I deemed it decorative. You can simply use aria-hidden=”true” on the image but still keep the title attribute which may be useful to sighted users.

## How to reproduce

Use clr-stack-view element and check its expand/collapse icon accessibility properties. Screenshot attached.

## Expected behavior

To have enough information for screen readers without repetition. 

## Versions

* Angular: 9
* Node: 10
* Clarity: 3

**Device:**

* Type: Desktop
* OS: Win10
* Browser Chrome
* Version 81

## Additional notes
Screenshot of the accessibility properties mentioned in this issue:
![expand_accordion](https://user-images.githubusercontent.com/43816378/80707765-2a532700-8af3-11ea-99ca-0fe42230f6f3.png)

**Comment by coryrylan** [comment]
This is following our guidelines provided by the a11y team. Will defer to @chlane

**Comment by clane** [comment]
I agree, In this case, the aria-label value is duplicative. The icon itself is visually duplicative since sighted users can see that the expansion has occurred without it.  Icons (images) only need accessible names when they provide meaning on their own, independent of any nearby textual content. If not they should be hidden from assistive technologies.

**Comment by clane** [comment]
We should remove the title attribute because it will create a tooltip that is not keyboard accessible

**Comment by sis0k0** [comment]
I can't seem to reproduce the issue. I've inspected the accordion component in an application here and the aforementioned title attribute is not present - https://stackblitz.com/edit/clarity-v4-accordion-accessibility?file=src/app/app.component.html. Could you please provide a reproduction?

**Comment by immihaylova** [comment]
@sis0k0 - clr-stack-view is the right component. The description is incorrect (good thing I added a screenshot). 
Sorry for the confusion.

**Comment by sis0k0** [comment]
Thank you for the prompt responses!
